lib/ssh/test/ssh_agent_SUITE.erl

```diff
@@ -39,18 +39,14 @@ all() ->
 
 init_per_suite(Config) ->
     ok = ssh:start(),
-    TempDir = filename:join(["", "tmp", ?MODULE_STRING]),
-    ok = file:make_dir(TempDir),
-    put_temp_dir(Config, TempDir).
+    Config.
 
-end_per_suite(Config) ->
-    TempDir = get_temp_dir(Config),
-    ok = file:del_dir(TempDir),
+end_per_suite(_Config) ->
     ok = ssh:stop().
 
 init_per_testcase(_TestCase, Config) ->
-    TempDir = get_temp_dir(Config),
-    PathPrefix = filename:join(TempDir, "agent."),
+    PrivDir = proplists:get_value(priv_dir, Config),
```

Comment:
When I tried to switch to using the Common Test `priv_dir`, the test stopped working. Somehow the socket of the mock agent was no longer created properly. The `priv_dir` seemed like a good place for the temporary mock agent socket and I expected the path not to make a difference.

But something seems to be different between using `/tmp` and `priv_dir` 🤔 Any ideas?